### PR TITLE
[New Term Entry] Java Math Methods: `.rint()`

### LINKED ISSUE #910: [Term Entry] Java Math Methods: `.rint()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.rint()` Java method.

- In the `terms/` folder, make a new folder called `rint`
- Inside the `rint` folder, create a new markdown file called `rint.md`.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by jcesardv**
### Description

Closes #910 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+Title: '.rint()'
+Description: 'Returns the closest double value depending on the value of the argument.'
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/rint/rint.md:3]:
```suggestion
Description: 'Returns the closest whole double value to the value of the argument.'
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.rint()`** returns the closest double value depending on the value of the argument.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/rint/rint.md:15]:
```suggestion
The **`Math.rint()`** returns the closest whole double value to the value of the argument.
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+```pseudo
+Math.rint(num)
+```
+
+The return type of the `.rint()` method and data type of the `num` parameter is a double.
+
+Some edge cases for the `.rint()` method include the following:
+
+- If the value of `num` is equal to an integer, then the result of the method is the value of `num`.
+- If the value of `num` is NaN or infinity, then the result of the method is the value of `num`.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/rint/rint.md:28]:
```suggestion
- If the value of `num` is `NaN` or infinity, then the result of the method is the value of `num`.
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+The **`Math.rint()`** returns the closest double value depending on the value of the argument.
+
+## Syntax
+
+```pseudo
+Math.rint(num)
+```
+
+The return type of the `.rint()` method and data type of the `num` parameter is a double.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/rint/rint.md:23]: This sentence is a little clumsy. You could rewrite it as something like: "The `.rint()` method takes a double as an argument and returns a double."

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jcesardv I did a first review and left a few comments.

File: content/java/concepts/math-methods/terms/rint/rint.md (hunk 4 above)
Comment (jcesardv) [reply content/java/concepts/math-methods/terms/rint/rint.md]: @SSwiniarski, done.

**Review by SSwiniarski — APPROVED** [review]
@jcesardv, looks like we're ready for a second review.

**Comment by jcesardv** [comment]
@SSwiniarski thanks for the feedback.

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.rint()`** returns the closest whole double value to the value of the argument.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/rint/rint.md:15]:
```suggestion
The **`Math.rint()`** returns a double that is rounded to the closest whole integer.
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
+---
+Title: '.rint()'
+Description: 'Returns the closest whole double value to the value of the argument.'
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/rint/rint.md:3]:
```suggestion
Description: 'Returns a double that is rounded to the closest whole integer.'
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/rint/rint.md:9]:
```suggestion
  - 'Functions'
  - 'Methods'
  - 'Arithmetic'
  - 'Numbers'
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+The **`Math.rint()`** returns the closest whole double value to the value of the argument.
+
+## Syntax
+
+```pseudo
+Math.rint(num)
+```
+
+The `.rint()` method takes a double as an argument and returns a double.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/rint/rint.md:23]:
```suggestion
The `.rint()` method takes a double `num` parameter and returns a double.
```

In the context of describing the function/method definition in the "Syntax" section, let's use "parameter". Any other times when the function/method is used in an actual example, we should use "argument."

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+Some edge cases for the `.rint()` method include the following:
+
+- If the value of `num` is equal to an integer, then the result of the method is the value of `num`.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/rint/rint.md:27]:
```suggestion
- If the value of `num` is passed an integer, then it is returned as a double.
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+- If the value of `num` is `NaN` or infinity, then the result of the method is the value of `num`.
+- If the value of `num` is positive or negative zero, then the result of the method is zero.
+
+## Example
+
+The following example shows the application of the `.rint()` method:
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/rint/rint.md:33]:
```suggestion
The following example demonstrates the `.rint()` method:
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+```pseudo
+Math.rint(num)
+```
+
+The `.rint()` method takes a double as an argument and returns a double.
+
+Some edge cases for the `.rint()` method include the following:
+
+- If the value of `num` is equal to an integer, then the result of the method is the value of `num`.
+- If the value of `num` is `NaN` or infinity, then the result of the method is the value of `num`.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/rint/rint.md:28]:
```suggestion
- If the value of `num` is `NaN` or positive/negative infinity, then the value of `num` is returned.
```

File: content/java/concepts/math-methods/terms/rint/rint.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+- If the value of `num` is positive or negative zero, then the result of the method is zero.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/rint/rint.md:29]:
```suggestion
- If the value of `num` is positive/negative zero, then `0.0` is returned.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@jcesardv Finished with my review and left some additional comments/suggestions. 😄

**Comment by jcesardv** [comment]
@Dusch4593, done.

**Review by Dusch4593 — APPROVED** [review]
@jcesardv LGTM! 👍🏻

**Comment by jcesardv** [comment]
Yes, it is my profile. Thanks for the feedback @Dusch4593.

**Pull request merged**